### ISSUE TITLE
authentication prompt seems not to work

### ISSUE THREAD

**Issue by zoidberg77** [nextcloud/server#3158]
### Steps to reproduce
1. add a new User
2. The authentication prompt pops up
3. enter the admin password

### Expected behaviour
the authentication prompt should accept the password

### Actual behaviour
when i enter the password and click ok, the prompt just reapears and i cannot add any user

### Server configuration
**Operating system**:
Ubuntu 16.04
**Web server:**
Apache/2.4.18 (Ubuntu)
**Database:**
Mysql 5.7.16
**PHP version:**
7.0.13
**Nextcloud version:** (see Nextcloud admin page)
11.0.1 (stable)
**Updated from an older Nextcloud/ownCloud or fresh install:**
fresh install
**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.4.1
  - comments: 1.1.0
  - dav: 1.1.1
  - federatedfilesharing: 1.1.1
  - federation: 1.1.1
  - files: 1.6.1
  - files_pdfviewer: 1.0.1
  - files_sharing: 1.1.1
  - files_texteditor: 2.2
  - files_trashbin: 1.1.0
  - files_versions: 1.4.0
  - files_videoplayer: 1.0.0
  - firstrunwizard: 2.0
  - gallery: 16.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - nextcloud_announcements: 1.0
  - notifications: 1.0.1
  - password_policy: 1.1.0
  - provisioning_api: 1.1.0
  - serverinfo: 1.1.1
  - sharebymail: 1.0.1
  - survey_client: 0.1.5
  - systemtags: 1.1.3
  - theming: 1.1.1
  - twofactor_backupcodes: 1.0.0
  - updatenotification: 1.1.1
  - workflowengine: 1.1.1
Disabled:
  - admin_audit
  - encryption
  - external
  - files_accesscontrol
  - files_automatedtagging
  - files_external
  - files_retention
  - templateeditor
  - user_external
  - user_ldap
  - user_saml

```
</details>

**The content of config/config.php:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "oc8oces0dcep",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "www.mydomain.com"
        ],
        "datadirectory": "\/var\/www\/nextcloud\/data",
        "overwrite.cli.url": "https:\/\/www.mydomain.com\/nextcloud",
        "dbtype": "mysql",
        "version": "11.0.1.2",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true
    }
}

```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Chrome 55
**Operating system:**
Ubuntu 16.04
#### Web server error log
<details>
<summary>Web server error log</summary>
```
[Thu Jan 19 14:08:17.093352 2017] [authz_core:error] [pid 430] [client 212.183.24.164:43623] AH01630: client denied by server configuration: /var/www/nextcloud/data/.ocdata
[Thu Jan 19 14:08:31.343276 2017] [authz_core:error] [pid 495] [client 212.183.24.164:43681] AH01630: client denied by server configuration: /var/www/nextcloud/data/.ocdata
[Thu Jan 19 14:11:29.735603 2017] [:error] [pid 499] [client 66.249.65.46:52672] PHP Deprecated:  The third argument to Zend\\Stratigility\\MiddlewarePipe() ($out) will be required starting with Stratigi$
[Thu Jan 19 14:16:16.020964 2017] [:error] [pid 427] [client 66.249.65.47:51892] PHP Deprecated:  The third argument to Zend\\Stratigility\\MiddlewarePipe() ($out) will be required starting with Stratigi$
[Thu Jan 19 14:21:19.704668 2017] [:error] [pid 500] [client 66.249.65.46:35174] PHP Deprecated:  The third argument to Zend\\Stratigility\\MiddlewarePipe() ($out) will be required starting with Stratigi$
[Thu Jan 19 14:22:26.368229 2017] [authz_core:error] [pid 500] [client 212.183.24.164:25514] AH01630: client denied by server configuration: /var/www/nextcloud/data/.ocdata
[Thu Jan 19 14:26:25.464526 2017] [:error] [pid 500] [client 66.249.65.44:36910] PHP Deprecated:  The third argument to Zend\\Stratigility\\MiddlewarePipe() ($out) will be required starting with Stratigi$
[Thu Jan 19 14:29:03.627299 2017] [authz_core:error] [pid 504] [client 212.183.24.164:26563] AH01630: client denied by server configuration: /var/www/nextcloud/data/.ocdata

```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
```
Nothing in there
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>
```
Missing argument $row in OC.Notification.hide() call, caller needs to be adjusted to only dismiss its own notification
```
</details>

**Comment by nickvergessen** [comment]
Works fine here. There is an [issue](https://github.com/nextcloud/server/pull/2953) with external user backend that is going to be fixed with 11.0.2

**Comment by zoidberg77** [comment]
i reinstalled nextcloud and now it seems to work, dont know what caused the issue
